vars/shared-OpenData.var.yml

```diff
@@ -63,7 +63,7 @@ extensions:
       description: "CKAN Extension for validating Data Packages"
       type: "git"
       url: "https://github.com/qld-gov-au/ckanext-validation.git"
-      version: "v0.0.8-qgov.6"
```

Comment:
Still need the last plugin updates included before we cut a staging release.

Reply:
That's what this is. https://github.com/qld-gov-au/ckanext-validation/releases/tag/v0.0.8-qgov.7